repair/row_level.cc

```diff
@@ -2948,7 +2949,7 @@ class row_level_repair {
                 if (table_dropped) {
                     throw replica::no_such_column_family(_shard_task.get_keyspace(),  _cf_name);
                 } else {
-                    throw std::runtime_error(format("Failed to repair for keyspace={}, cf={}, range={}", _shard_task.get_keyspace(), _cf_name, _range));
+                    throw std::runtime_error(format("Failed to repair for keyspace={}, cf={}, range={}, ex={}", _shard_task.get_keyspace(), _cf_name, _range, ex));
```

Comment:
[std::throw_with_nested](https://en.cppreference.com/w/cpp/error/throw_with_nested)?

Reply:
It needs to be called inside `catch`.  Is it worth rethrowing?

Reply:
Oh, I didn't know seastar has it's version of nested_exception! That should make it